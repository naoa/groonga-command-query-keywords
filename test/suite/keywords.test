plugin_register commands/query_keywords

table_create Entries TABLE_NO_KEY
column_create Entries title COLUMN_SCALAR ShortText

query_keywords Entries "hoge + test OR or - NOT (kakko + kakko2) + title:@'huga'"
