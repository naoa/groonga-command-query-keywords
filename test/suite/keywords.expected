plugin_register commands/query_keywords
[[0,0.0,0.0],true]
table_create Entries TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Entries title COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
query_keywords Entries "hoge + test OR or - NOT (kakko + kakko2) + title:@'huga'"
[[0,0.0,0.0],["'huga'","kakko2","kakko","or","test","hoge"]]
